movai_core_shared/common/config.py

```diff
@@ -0,0 +1,33 @@
+"""
+   Copyright (C) Mov.ai  - All Rights Reserved
+   Unauthorized copying of this file, via any medium is strictly prohibited
+   Proprietary and confidential
+
+   Usage:
+        Holds various general functions related to robot configuration.
+
+   Developers:
+   - Erez Zomer (erez@mov.ai) - 2022
+"""
+from movai_core_shared.envvars import REDIS_MASTER_HOST
+
+def is_manager() -> bool:
+    """Tell the process if its running on the manager host machine.
+
+    Returns:
+        bool: true if the fuction runs on a manager host machine, false otherwise.
+    """
+    return REDIS_MASTER_HOST == "redis-master"
```

Comment:
can be none?

Reply:
it does not need to be but it could be none in case the environment variable does not exist, 
we need to change [this](https://github.com/MOV-AI/movai-core-shared/blob/5fa4f67165dbb2c15885f2a1b190e7fc101ecb8f/movai_core_shared/envvars.py#L34) line in order to cover this
change to "REDIS_MASTER_HOST = os.getenv("REDIS_MASTER_HOST", "redis-master")"